### PR TITLE
Fix the security tasks.

### PR DESCRIPTION
**Pull request by myidpt**

### CODE REVIEW

File: content/docs/tasks/security/health-check/index.md [hunk 1]
```diff
@@ -7,9 +7,11 @@ keywords: [security,health-check]
 
 You can enable Citadel's health checking feature
 to detect the failures of the Citadel CSR (Certificate Signing Request) service.
-Citadel periodically sends CSRs to its CSR service and verifies the response.
+When a failure is detected, Kubelet will be automatically start the Citadel container.
```
Comment (geeknoid) [inline content/docs/tasks/security/health-check/index.md]: will be automatically start -> will automatically restart

File: content/docs/tasks/security/health-check/index.md [hunk 2]
```diff
@@ -41,13 +43,16 @@ See the [authentication policy task](/docs/tasks/security/authn-policy/) for det
...
 {{< text bash >}}
-$ kubectl apply -f install/kubernetes/istio-citadel-with-health-check.yaml
+$ helm template install/kubernetes/helm/istio --name istio --namespace istio-system \
+-x charts/security/templates/deployment.yaml \
+--set global.mtls.enabled=true --set security.citadelHealthCheck=true > citadel-health-check.yaml
+$ kubectl apply -f citadel-health-check.yaml
 {{< /text >}}
 
-## Verifying the health checker is working
+## Verifying the health checking is working
```
Comment (geeknoid) [inline content/docs/tasks/security/health-check/index.md]: Verifying that health checking is working

File: content/docs/tasks/security/health-check/index.md (hunk 1 above)
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
When a failure is detected, Kubelet automatically starts the Citadel container.
```

File: content/docs/tasks/security/health-check/index.md [hunk 3]
```diff
@@ -7,9 +7,11 @@ keywords: [security,health-check]
 
 You can enable Citadel's health checking feature
 to detect the failures of the Citadel CSR (Certificate Signing Request) service.
-Citadel periodically sends CSRs to its CSR service and verifies the response.
+When a failure is detected, Kubelet will be automatically start the Citadel container.
 
-The _prober client_ module in Citadel periodically checks the health status of Citadel's CSR gRPC server.
+When the health checking feature is enabled,
+the _prober client_ module in Citadel periodically checks the health status of Citadel's CSR gRPC server,
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
the **prober client** module in Citadel periodically checks the health status of Citadel's CSR gRPC server,
```

File: content/docs/tasks/security/health-check/index.md [hunk 4]
```diff
@@ -7,9 +7,11 @@ keywords: [security,health-check]
...
+by sending CSRs to the gRPC server and verifying the response.
```
Comment (rcaballeromx) [inline content/docs/tasks/security/health-check/index.md]:
```suggestion
sends CSRs to the gRPC server, and verifies the response.
```

File: content/docs/tasks/security/health-check/index.md (hunk 2 above)
Comment (rcaballeromx) [reply content/docs/tasks/security/health-check/index.md]:
```suggestion
## Verify that health checking works
```

File: content/docs/tasks/security/health-check/index.md (hunk 1 above)
Comment (myidpt) [reply content/docs/tasks/security/health-check/index.md]: Done.

**Comment by myidpt** [comment]
ci/circleci is failing due to unrelated broken external links.

**Comment by rcaballeromx** [comment]
@myidpt I triggered a rebuild to see if that solves the linting issue. Once it passes, I'll approve.

**Pull request merged**